.travis.yml

```diff
@@ -1,30 +1,24 @@
 language: java
 sudo: false
-
 services:
-  - redis-server
-
+- redis-server
```

Comment:
Is all the white-space change intentional? It's kind of hard to find the real changes :(